### PR TITLE
Rapid 'response page' requests from a single student #3012

### PR DESCRIPTION
**Pull request by thyageshm**
@damithc I have ran StudentFeedbackSubmitPageUiTest, StudentFeedbackSubmissionEditPageAction, and AllAccessControlUiTests. The testing station is currently running the AllTests

### CODE REVIEW

File: src/main/java/teammates/ui/controller/StudentFeedbackSubmissionEditPageAction.java [hunk 1]
```diff
@@ -125,4 +125,9 @@ protected StudentAttributes getStudent() {
         
         return student;
     }
+    
+    protected boolean isRegisteredStudent(){
+        // a registered student must have an associated google Id
+        return (account.googleId != null) && (!account.googleId.isEmpty());
+    }
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/StudentFeedbackSubmissionEditPageAction.java]: Should this method be in the account object instead? Something like `isRegistered():boolean`?

File: src/main/webapp/entityNotFoundPage.jsp [hunk 2]
```diff
@@ -22,12 +22,30 @@
...
         <div class="row">
-            <div class="alert alert-warning col-md-4 col-md-offset-4">
+            <div class="alert alert-warning col-md-6 col-md-offset-3">
                 <img src="/images/puzzled.png"
                     style="float: left; margin: 0 10px 10px 0; height: 90px;">
                 <p>
-                    We could not locate what you were trying to access.<br><br>
-                    Make sure that the URL is correct, and you are a registered user of TEAMMATES. [<a href="/logout.jsp">Logout</a>]
+                    TEAMMATES could not locate what you were trying to access. <br><br>
+                    If you are a registered user you can go back to the <a href="/page/studentHomePage">home page</a>
```
Comment (damithc) [inline src/main/webapp/entityNotFoundPage.jsp]: Inserting this sentence here breaks the link between the one above and the one below.

**Comment by damithc** [comment]
Some minor tweaks. Can merge.

**Comment by thyageshm** [comment]
changes have been made, I am proceeding to merge

**Comment by thyageshm** [comment]
fix has been merged

**Pull request merged**